modules/ROOT/pages/batch-processing.adoc

```diff
@@ -600,12 +603,28 @@ image::example-mapping.png[example_mapping]
 [discrete.view]
 === Studio Visual Editor
 
-image::example-query3.png[example_query3]
+
+image::mruntime-batch-example-3.png[example_query3]
+
+** The first item shows the MEL expression that evaluates against the message payload, which in batch processing is the payload of the record.
+
+
+image::mruntime-batch-example-4.png[Message enricher scope]
+
+
+** The second item shows the Message Enricher component view. This component enriches rather than manipulates the payload of the record.
```

Comment:
I don't understand the purpose of this sentence: "This component enriches rather than manipulates the payload of the record." 
Would it be better to just say "This component enriches the payload of the record." ?